webhook/core/pod_webhook_test.go

```diff
@@ -153,6 +174,15 @@ func TestPodResourceInjector_Handle(t *testing.T) {
 	}
 }
 
+// TestFargatePodSG_Handle test webhook annotating fargate pod with sg list.
+func TestFargatePodSG_Handle(t *testing.T) {
+	pod := NewFargatePod("test", "test_fp", "test_namespace")
+	assert.True(t, pod.ObjectMeta.Labels["eks.amazonaws.com/fargate-profile"] == "test_fp")
+	injector := getFargatePodResourceInjectorWithSGP()
+	resp := getResponse(pod, injector)
+	assert.True(t, resp.Allowed)
```

Comment:
Can we also verify the pod got annotated with the expected SG.

Reply:
If possible, could you also add test cases for following scenarios.

- If fargate pod's annotation is updated outside the mutating webhook. --> It should be denied
- If fargate pod is created with the sgp annotation. --> It should be denied
- All non fargate pod should never have this annotation.